### PR TITLE
cleanup

### PR DESCRIPTION
**Pull request by turadg**
I was looking something up with governance and noticed some governance types where `unknown` here so this adds a commit to import them.

While poking around I noticed the `walletAtom` never has a setter so it's always null. I deleted all code that only runs when it's not null, which meant a lot of other atoms were unused, so this deletes those too.

### CODE REVIEW

File: src/store/app.ts [hunk 1]
```diff
@@ -11,16 +9,8 @@ export type BrandInfo = DisplayInfo<'nat'> & {
...
-export const walletAtom = atom<ERef<any>>(null);
-
-export const offersAtom = atom<Array<any> | null>(null);
-
-export const pursesAtom = atom<Array<PursesJSONState> | null>(null);
-
-/** A map of anchor brand petnames to their instance ids. */
-export const instanceIdsAtom = mapAtom<string, string>();
+// XXX never filled so all the display functions are generic
+const brandToInfoAtom = mapAtom<Brand, BrandInfo>();
```
Comment (samsiegart) [inline src/store/app.ts:13]: So any code that uses display functions is broken? We should rip out display functions and all that code to be safe

File: src/utils/updates.ts [hunk 2]
```diff
@@ -1,148 +0,0 @@
-import { iterateLatest, makeFollower, makeLeader } from '@agoric/casting';
```
Comment (samsiegart) [inline src/utils/updates.ts:1]: I guess this is just because we hard forked dapp-PSM

**Review by samsiegart — CHANGES_REQUESTED** [review]

File: src/utils/updates.ts (hunk 2 above)
Comment (turadg) [reply src/utils/updates.ts:1]: exactly. then started getting data directly from storage paths with hooks

File: src/store/app.ts (hunk 1 above)
Comment (turadg) [reply src/store/app.ts:13]: Not broken, IIUC. They just operate without the brand info map. e.g. getDecimalPlaces reads from `wellKnownBrands`. `getPetname` is always empty string.

There is more tech debt to burn down here but that needn't block the improvements made by this PR. This PR as is cuts a lot of dead code and lights the way to more tech debt that can be removed.

**Review by samsiegart — APPROVED** [review]

**Pull request merged**